### DISCUSSION TITLE
How can I add a comment without any prompt

### DISCUSSION THREAD

**Discussion by namp10010** [ankitpokhrel/jira-cli#627] [Q&A]
I am trying to run jira-cli inside a github action and this is the current error

```
ghcr.io/ankitpokhrel/jira-cli:latest issue comment add GHIN-1 "hello world from github actions"
25l? What's next?  [Use arrows to move, type to filter]
> Submit
```

Is there anyway to just add a comment without any prompt?

On local machine I was able to achieve that by 

```bash
echo "hello" | jira issue comment add GHIN-1
```

However, as the github action runs the docker container with default entrypoint `/bin/jira` it will be too much hassle to change the entrypoint to achieve something that should be quite obvious.

In general, is there a mode where no prompt is required so that jira-cli is more friendly with CI/CD automation?

Any help is much appreciated.

**Comment by namp10010** [comment]
I found a way to achieve this, not sure if this is the intended way though. Feel free to close the discussion if that's the case

```
ghcr.io/ankitpokhrel/jira-cli:latest issue comment add GHIN-1 --template - "hello world from github actions"
```

**Comment by ankitpokhrel** [comment]
Hi @namp10010, there is a `--no-input` option available or you can get input from stdin (like you did) which will skip the UI by default.

```sh
jira issue comment add GHIN-1 "hello world from github actions" --no-input
```
